Cannot compile project that uses @ant-design/pro-components

**Steps to reproduce:**
1. Compile a project that uses ProList

`webpack 5.75.0 compiled with 5 errors`

`ERROR in ./node_modules/@ant-design/pro-field/es/components/DatePicker/index.js 9:0-49
Module not found: Error: Can't resolve 'dayjs/plugin/weekOfYear'`

`ERROR in ./node_modules/@ant-design/pro-field/es/components/FromNow/index.js 7:0-53
Module not found: Error: Can't resolve 'dayjs/plugin/relativeTime'`

`ERROR in ./node_modules/@ant-design/pro-field/es/index.js 42:0-43
Module not found: Error: Can't resolve 'dayjs/plugin/weekday'`

`ERROR in ./node_modules/@ant-design/pro-field/es/index.js 43:0-49
Module not found: Error: Can't resolve 'dayjs/plugin/localeData'`

`ERROR in ./node_modules/@ant-design/pro-utils/es/conversionMomentValue/index.js 5:0-55
Module not found: Error: Can't resolve 'dayjs/plugin/quarterOfYear'`

--- a/index.js
+++ b/index.js
@@ -4,14 +4,20 @@ const updateResource = (resource) => {
 };
 const plugin = 'AntdMomentWebpackPlugin';
 class Plugin {
+  constructor (options) {
+    this.options = options || {};
+  }
   apply(compiler) {
+    const { disableDayjsAlias } = this.options;
     const { alias } = compiler.options.resolve;
-    if (alias) {
-      alias.dayjs = 'moment';
-    } else {
-      compiler.options.resolve.alias = {
-        dayjs: 'moment',
-      };
+    if (!disableDayjsAlias) {
+      if (alias) {
+        alias.dayjs = 'moment';
+      } else {
+        compiler.options.resolve.alias = {
+          dayjs: 'moment',
+        };
+      }
     }
     compiler.hooks.normalModuleFactory.tap(plugin, (factory) => {
       factory.hooks.beforeResolve.tap(plugin, (result) => {

